src/libs/ReportUtils.js

```diff
@@ -460,6 +460,17 @@ function isConciergeChatReport(report) {
     return lodashGet(report, 'participantAccountIDs', []).length === 1 && Number(report.participantAccountIDs[0]) === CONST.ACCOUNT_ID.CONCIERGE && !isChatThread(report);
 }
 
+/**
+ * Returns true if we can share a task in this report
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+
+function canShareTaskInReport(report) {
```

Comment:
This can be general function, not tied to task.
So let's rename it to `isExpensifyChatReport`

Reply:
Done, thanks 🙏